rpm/cray/csm/sle-15sp3/index.yaml

```diff
@@ -23,8 +23,18 @@
 #
 https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp3/:
   rpms:
-    - cray-site-init-1.32.0-1.x86_64
-    - craycli-0.82.8-1.x86_64
+    - cfs-debugger-1.4.0-1.noarch
+    - cfs-state-reporter-1.9.3-1.noarch
+    - cfs-trust-1.6.7-1.noarch
+    - hpe-yq-4.33.3-1.aarch64
     - hpe-yq-4.33.3-1.x86_64
     - libcsm-0.0.4-1.noarch
-    - loftsman-1.2.0-2.x86_64
+    - loftsman-1.2.0-3.x86_64
+
+# Add some noos RPMs into this SLE repo to avoid breaking recipes which may
+# expect them to be found here
+https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/noos/:
+  rpms:
+    - cray-site-init-1.32.0-1.x86_64
+    - craycli-0.82.9-1.aarch64
+    - craycli-0.82.9-1.x86_64
```

Comment:
cray-site-init currently publishes to sp3, sp4, and noos to prevent this. Please leave it in the sp3 repo list at the top.

I would appose mix-n-matching the repo files, the file is sle-15sp3 and it should only have sle-15sp3 repos defined in it.

What other recipes are we worried about here? csm-config already adds noos, it will have craycli available to it.

Reply:
I don't know which recipes may use these. I just know that when we contemplated converting the CMS repos to noos, David Laine noted that this caused problems because of product recipes relying on RPMs being in particular repos.

I figure keeping this section in the SLE repos just for CSM 1.5 is a reasonable compromise until they are fully moved to the noos repo in CSM 1.6.